addons/mail/models/mail_channel_partner.py

```diff
@@ -23,6 +23,7 @@ class ChannelPartner(models.Model):
     fold_state = fields.Selection([('open', 'Open'), ('folded', 'Folded'), ('closed', 'Closed')], string='Conversation Fold State', default='open')
     is_minimized = fields.Boolean("Conversation is minimized")
     is_pinned = fields.Boolean("Is pinned on the interface", default=True)
+    last_activity_time = fields.Datetime('Last activity time', default=fields.Datetime.now)
```

Comment:
This default will not work, it will have the time when the server started, you need a lambda for it.